bundle/EventListener/ScheduledVisibilityListener.php

```diff
@@ -30,6 +33,19 @@ public static function getSubscribedEvents(): array
     public function onPublishVersion(PublishVersionEvent $event): void
     {
         $content = $event->getContent();
+
+        if (!$this->configurationService->isEnabled()) {
+            return;
+        }
+
+        $allowedAll = $this->configurationService->isAllContentTypes();
+        $allowedContentTypes = $this->configurationService->getAllowedContentTypes();
+
+        $contentType = $content->getContentType();
+        if (!$allowedAll && !in_array($contentType, $allowedContentTypes, true)) {
```

Comment:
You can simplify this by adding `isContentTypeAllowed()` to the `ScheduledVisibilityConfiguration` service.

Reply:
fixed here f8d6b29acd54f731059dc8401498ed5c6348766f